source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -3841,21 +3841,19 @@ class ActionButton {
 
 class ActionButtonWithTelemetry extends ActionButton {
 	
-	render(alignment: Enums.ActionAlignment) {
-
-		var useGUID = GUIDHelper.getOrCreate().isGUIDtracked();
-		var guid = GUIDHelper.getOrCreate().getGUID().toString();
+    render(alignment: Enums.ActionAlignment) {
+        let useGUID = GUIDHelper.getOrCreate().isGUIDtracked();
```

Comment:
More on this - applications usually render multiple different cards. All these cards will share the same GUID given it's global. That doesn't seem like that's what you're trying to accomplish.